packages/@aws-cdk/aws-apigateway/lib/resource.ts

```diff
@@ -140,7 +140,7 @@ function validateResourcePathPart(part: string) {
         part = part.substr(1, part.length - 2);
     }
 
-    if (!/^[a-zA-Z0-9\.\_\-]+$/.test(part)) {
+    if (!/^[a-zA-Z0-9\.\_\-]+[\+]?$/.test(part)) {
         throw new Error(`Resource's path part only allow a-zA-Z0-9._- and curly braces at the beginning and the end: ${part}`);
```

Comment:
Can you please update the message?

Reply:
@eladb I updated the message to account for the optional trailing '+'.